app/models/employee.rb

```diff
@@ -1,16 +1,16 @@
 # frozen_string_literal: true
 
 class Employee < ApplicationRecord
-  # Include default devise modules. Others available are:
-  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
   devise :database_authenticatable, :registerable,
          :recoverable, :rememberable, :validatable
 
   has_many :given_kudos, class_name: 'Kudo', foreign_key: 'giver_id', dependent: :destroy, inverse_of: :giver
   has_many :received_kudos, class_name: 'Kudo', foreign_key: 'receiver_id', dependent: :destroy, inverse_of: :receiver
-  has_many :orders, dependent: :nullify
+  has_many :orders, dependent: :destroy
```

Comment:
why change to destroy?

Reply:
because there is not null constrain in db, and i was thinking what should i change. I think that if we don't have employee we do not need to keep his orders.